No configuration file found.
No pyproject.toml file found.
stubPath /mnt/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/typings is not a valid directory.
Assuming Python platform Linux
Searching for source files
Found 7 source files
/home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py:30:6 - error: Import "tfx.dsl.io" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py:31:6 - error: Import "tfx.proto" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py:32:6 - error: Import "tfx.proto" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py:33:6 - error: Import "tfx.utils" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py:154:9 - error: Argument of type "Unknown | Dict[Text, Any]" cannot be assigned to parameter "message" of type "Message" in function "MessageToDict"
    Type "Unknown | Dict[Text, Any]" cannot be assigned to type "Message"
      "Dict[Text, Any]" is incompatible with "Message" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py:159:9 - error: Argument of type "Unknown | Dict[Text, Any]" cannot be assigned to parameter "message" of type "Message" in function "MessageToDict"
    Type "Unknown | Dict[Text, Any]" cannot be assigned to type "Message"
      "Dict[Text, Any]" is incompatible with "Message" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py:212:9 - error: Argument of type "Unknown | Dict[Text, Any]" cannot be assigned to parameter "message" of type "Message" in function "MessageToDict"
    Type "Unknown | Dict[Text, Any]" cannot be assigned to type "Message"
      "Dict[Text, Any]" is incompatible with "Message" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py:450:26 - error: No overloads for "sub" match the provided arguments
    Argument types: (str, Text | List[Text], str) (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py:571:35 - error: Operator ">" not supported for "None" (reportOptionalOperand)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py:578:38 - error: Operator ">=" not supported for types "int | None" and "int | None"
    Operator ">=" not supported for types "int" and "None"
    Operator ">=" not supported for types "None" and "None"
    Operator ">=" not supported for types "None" and "int" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py:588:45 - error: Object of type "None" is not subscriptable (reportOptionalSubscript)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py:591:24 - error: No overloads for "__new__" match the provided arguments
    Argument types: (list[str], List[Text] | None) (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/example_gen/utils.py:595:21 - error: No overloads for "sub" match the provided arguments
    Argument types: (str, Text | None, str | Unknown) (reportGeneralTypeIssues)
/home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:27:6 - error: Import "tfx" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:28:6 - error: Import "tfx.components.infra_validator" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:29:6 - error: Import "tfx.components.util" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:30:6 - error: Import "tfx.components.util" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:31:6 - error: Import "tfx.dsl.io" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:32:6 - error: Import "tfx.proto" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:33:6 - error: Import "tfx.proto" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:34:6 - error: Import "tfx.types" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:35:6 - error: Import "tfx.utils" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:36:6 - error: Import "tfx_bsl.tfxio" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:39:6 - error: Import "tensorflow_serving.apis" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:40:6 - error: Import "tensorflow_serving.apis" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:41:6 - error: Import "tensorflow_serving.apis" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:140:27 - error: Expected class type but received "type" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:140:27 - error: Argument to class must be a base class (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/components/infra_validator/request_builder.py:183:20 - error: No overloads for "join" match the provided arguments
    Argument types: (Unknown, Text | None, Literal['*']) (reportGeneralTypeIssues)
/home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/fileio.py
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/fileio.py:23:6 - error: Import "tfx.dsl.io" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/fileio.py:24:6 - error: Import "tfx.dsl.io" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/fileio.py:25:6 - error: Import "tfx.dsl.io.filesystem" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/fileio.py:28:8 - error: Import "tfx.dsl.io.plugins.local" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/fileio.py:29:8 - error: Import "tfx.dsl.io.plugins.tensorflow_gfile" could not be resolved (reportMissingImports)
/home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/local.py
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/local.py:27:6 - error: Import "tfx.dsl.io" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/local.py:28:6 - error: Import "tfx.dsl.io" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/local.py:29:6 - error: Import "tfx.dsl.io.filesystem" could not be resolved (reportMissingImports)
/home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:23:6 - error: Import "tfx.dsl.io" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:24:6 - error: Import "tfx.dsl.io" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:25:6 - error: Import "tfx.dsl.io.filesystem" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:41:17 - error: "io" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:45:10 - error: "io" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:49:17 - error: "io" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:53:17 - error: "io" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:57:17 - error: "io" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:61:17 - error: "io" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:65:10 - error: "io" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:69:10 - error: "io" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:73:10 - error: "io" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:77:10 - error: "io" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:81:10 - error: "io" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:85:17 - error: "io" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:93:21 - error: "io" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:98:26 - error: Expression of type "None" cannot be assigned to declared type "Type[TensorflowFilesystem]"
    Type "None" cannot be assigned to type "Type[TensorflowFilesystem]" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/dsl/io/plugins/tensorflow_gfile.py:98:3 - error: Declaration "TensorflowFilesystem" is obscured by a declaration of the same name (reportGeneralTypeIssues)
/home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/orchestration/portable/python_executor_operator.py
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/orchestration/portable/python_executor_operator.py:19:6 - error: Import "tfx" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/orchestration/portable/python_executor_operator.py:20:6 - error: Import "tfx.dsl.components.base" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/orchestration/portable/python_executor_operator.py:21:6 - error: Import "tfx.dsl.io" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/orchestration/portable/python_executor_operator.py:22:6 - error: Import "tfx.orchestration.portable" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/orchestration/portable/python_executor_operator.py:23:6 - error: Import "tfx.orchestration.portable" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/orchestration/portable/python_executor_operator.py:24:6 - error: Import "tfx.proto.orchestration" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/orchestration/portable/python_executor_operator.py:25:6 - error: Import "tfx.proto.orchestration" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/orchestration/portable/python_executor_operator.py:26:6 - error: Import "tfx.types.value_artifact" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/d14aa7086f05b696a68db3a932cc750a333c8567/a/tfx/orchestration/portable/python_executor_operator.py:27:6 - error: Import "tfx.utils" could not be resolved (reportMissingImports)
64 errors, 0 warnings, 0 infos 
Completed in 2.331sec

time cost(in seconds): 2.593168020248413
